### ISSUE TITLE
"too few arguments" error when using a form containing a field named "format"

### ISSUE THREAD

**Issue by multimeric** [OSC/ondemand#3278]
If you have an interactive app with the following `form.yml`:

```yml
---
form:
  - cluster
  - format
```

It correctly renders like this:
![Screen Shot 2024-01-05 at 12 40 19 pm](https://github.com/OSC/ondemand/assets/5019367/9a9a95ed-8bfb-4156-8fc9-af72d0911054)

However, when you try to submit the job, the following error will *always* appear:
```
Failed to stage the template with the following error:
too few arguments
The Date Info session data for this session can be accessed under the staged root directory.
```

I was tearing my hair out for a few hours before I worked this out. I'm writing this issue to document this bug if nothing else, but also I'm a bit worried that this field name is being invoked as a function.

**Comment by johrstrom** [comment]
What's your `submit.yml.erb`?  Because this is happening during submission, that's where I'd look.

> but also I'm a bit worried that this field name is being invoked as a function.

This is good intuition, but I feel like we fixed that. I can confirm if it's 3.0.3 if you still have issues after we look at the submit.yml.erb.

**Comment by multimeric** [comment]
My submit.yml was the most minimal one and didn't seem to affect this issue. I can give you the file when I'm at my computer but I didn't get the sense that it mattered.

**Comment by johrstrom** [comment]
🤦‍♂️ I can replicate now after fixing up my test app. I'll look into it.

**Comment by johrstrom** [comment]
Yes it is thinking that `format` is a method that it's calling.  As a workaround you can call that field something else and apply a `label` to it that shows just format in the UI.

I'll keep this ticket open as it's similar to another we've had in the past.

**Comment by bstepanovski** [comment]
Addressed in PR for #5075
